src/Runners/PHPUnit/Options.php

```diff
@@ -1029,6 +1045,11 @@ public function coverageXml(): ?string
         return $this->coverageXml;
     }
 
+    public function noCoverage(): bool
```

Comment:
On a second analisys, this is not needed, since we only use `hasCoverage` method

Reply:
Solved by 4067542